.github/workflows/package.yml

```diff
@@ -54,6 +57,7 @@ jobs:
         run: cmake --build "$BUILD_DIR" --target package
 
       - name: Upload binaries to release
+        if: github.event_name != 'pull_request'
```

Comment:
Nitpick but I would prefer to have.
In case we introduce another event.

if: github.event_name == 'tags'

Reply:
I'm not sure I understand. You mean replacing this line with `if: github.event_name == 'tags'`? And what is 'tags' here a custom definition or a standard GitHub definition

Reply:
Sorry it should be 
if: github.ref_type == 'tag'

Reply:
Fixed with c428429